### ISSUE TITLE
docker_daemon check parses container names incorrectly

### ISSUE THREAD

**Issue by knspriggs** [DataDog/dd-agent#1991]
The docker daemon check seems to be parsing container names incorrectly. I am running the agent in a docker container on AWS ECS, so the ecs-agent sets my container names for me, ex. `ecs-service-name-20-componentA-8ebab182c`

In several circumstances the container names are jumbled together when they appear in datadog, ex:
```
ecs-service-name-20-componentA-sfda831bcdf0
ecs-service-name-20-componentA-aee1f3ffcd/componentB
ecs-service-name-20-componentD-c091fbcdf0
ecs-service-name-20-componentA-eba83d40300/componentC
```

Where I would expect them to be:
```
ecs-service-name-20-componentA-sfda831bcdf0
ecs-service-name-20-componentB-asfd71e9anw
ecs-service-name-20-componentD-c091fbcdf0
ecs-service-name-20-componentC-fas8wefnai
```

Other notes:
- these containers are pulled from a private registry, therefor the image name container they registry address.
- I am using the 5.5.1 release of the datadog agent.

**Comment by hkaj** [comment]
Hi @knspriggs, thank you for reaching out about this. This bug is due to a mis-handling of container aliases. We actually [fixed it recently](https://github.com/DataDog/dd-agent/commit/a20a2be098dde2d06f9ea7a83899fc9bbb85bac1) and the fix will go out with the 5.6 release (in a week or so).

Let me know if you have any other question.

**Comment by knspriggs** [comment]
Awesome, I'll keep an eye out for the next release. Cheers.
